pkg/aggregate/aggregate.go

```diff
@@ -223,3 +232,140 @@ func (a AggregatesList) Clear() {
 		aggr.Clear()
 	}
 }
+
+func GetHiddenAggregates(mask AggrType, requestedAggregates []AggrType) []AggrType {
```

Comment:
consider replacing `[]AggrType` with single aggregated `AggrType` (bitwise OR)
It will make code simpler and reduce complexity of checks